tools/monorepo-utils/src/code-freeze/commands/milestone/index.ts

```diff
@@ -32,7 +31,7 @@ export const milestoneCommand = new Command( 'milestone' )
 		'-m --milestone <milestone>',
 		'Milestone to create. Next milestone is gathered from Github if none is supplied'
 	)
-	.action( async ( options: Options ) => {
```

Comment:
This type isn't needed, its auto generated by `extra-typings` and extra typings will ensure it's always up to date and correct. ✨